### PR TITLE
chore: update metadata descriptions for static site resources

### PR DESCRIPTION
**Pull request by dannyrandall**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml
Comment (huanjani) [inline internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml:136]: here and throughout, for consistency with the other templates?
```suggestion
      'aws:copilot:description': 'A lambda that starts the process of moving files to the S3 bucket'
```

File: internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml
Comment (huanjani) [inline internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml]:
```suggestion
      'aws:copilot:description': 'An IAM Role for the lambda that starts the process of moving files to the S3 bucket'
```

File: internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml
Comment (huanjani) [inline internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml]:
```suggestion
      'aws:copilot:description': 'A custom resource that starts the process of moving files to the S3 bucket'
```

File: internal/pkg/template/templates/workloads/services/static-site/cf.yml
Comment (huanjani) [inline internal/pkg/template/templates/workloads/services/static-site/cf.yml]:
```suggestion
      'aws:copilot:description': 'An IAM Role for the lambda that starts the process of moving files to the S3 bucket'
```

File: internal/pkg/template/templates/workloads/services/static-site/cf.yml
Comment (huanjani) [inline internal/pkg/template/templates/workloads/services/static-site/cf.yml]:
```suggestion
      'aws:copilot:description': 'A custom resource that starts the process of moving files to the S3 bucket'
```

**Review by huanjani — COMMENTED** [review]
just nits

**Review by Lou1415926 — APPROVED** [review]
LGTM! But let's merge this after the other fixes are merged :cool-crying

**Review by bvtujo — APPROVED** [review]

**Pull request merged**
